caom2utils/caom2utils/fits2caom2.py

```diff
@@ -1747,9 +1751,12 @@ def _get_datetime(self, from_value):
                 return from_value
             else:
                 result = None
+                # CFHT 2003/03/29,01:34:54
```

Comment:
Clean up?

Reply:
It's meant as a comment to indicate which telescope originated which pattern. If you don't find that useful information, let me know, I'll take it out.